application/views/admin_lookup.php

```diff
@@ -39,8 +39,8 @@
             <th>证件编号</th>
             <th>北大赛</th>
             <th>住宿</th>
-            <th>5.14晚餐</th>
-            <th>5.15午餐</th>
+            <th>5.6晚餐</th>
```

Comment:
I think `5.6` style has no ambiguity with the date specified, which is better for comprehension. In fact we do not have races on the first day.